quoracle/quorum_system.py

```diff
@@ -520,11 +520,17 @@ def fr_load(problem: pulp.LpProblem, fr: float) -> pulp.LpAffineExpression:
 
                 if x in x_to_read_quorum_vars:
                     vs = x_to_read_quorum_vars[x]
-                    x_load += fr * sum(vs) / self.node(x).read_capacity
+                    xread_capacity =  self.node(x).read_capacity
+                    if xread_capacity is None:
```

Comment:
This code would not be necessary if we made:

```
read_capacity: float = 1.0
```

in the dataclass. More instances below.